index.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Maher</title>
+    <link rel="stylesheet" href="css/style.css">
+    <!-- link swiper css -->
+    <link rel="stylesheet" href="CSS/swiper-bundle.min.css">
+
+    <!-- BOX ICONS -->
+</head>
+
+    <!-- Header -->
+        <!-- nav -->
+        <header>
```

Comment:
- [ ] There should be a body tag here, just immediately after the head tag and the header tag would be inside the body tag.

Reply:
ok i got it , i will fix it , maybe I delette it accidently.